js/electron/transcoding.js

```diff
@@ -53,8 +127,48 @@ function transcodingFactory(electronIpcRenderer) {
       electronIpcRenderer.send('transcode-video-request', filePath);
     });
   }
+
+  return { downloadBinaries, transcode };
 }
 
+async function binariesDownloader(electronIpcMain, userDataFolder) {
```

Comment:
### FF Binaries download logic
Checks if there is free space for FFMPEG and FFPROBE. Assuming that we need at least 1 Gb for stable work of app.

This function is listening to events called from UI scripts too.